packages/sit-onyx/src/utils/playwright.tsx

```diff
@@ -109,7 +109,7 @@ export const executeScreenshotsForAllStates = <
   // therefore, we will split the permutations to only run a maximum of X permutations inside a single test
   const batches: (typeof permutations)[] = [];
   while (permutations.length > 0) {
-    batches.push(permutations.splice(0, 100));
+    batches.push(permutations.splice(0, 50));
```

Comment:
```suggestion
    batches.push(permutations.splice(0, 100));
```

Lets change this back, I did this change when I tried to fix the pipeline but since it had no effect we can revert it :)